process-engine/bpmn-studio
# ♻️Refactor Property Panel to use new Elements Contracts

**Changes:**

1. Update the `@process-engine/bpmn-elements_contracts` dependency.
2. Refactor the Property Panel sections accordingly.

PR: #PullRequest

## How can others test the changes?

1. Start BPMN-Studio
2. Open any diagram
3. See that anything works as expected.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
